admin/src/create/CardUpload.tsx

```diff
@@ -0,0 +1,31 @@
+import { FileField, FileInput, SimpleForm } from 'react-admin';
+import { FieldValues } from 'react-hook-form';
+
+export function CardUpload(): JSX.Element {
+  const uploadCards = (data: FieldValues) => {
+    const formData = new FormData();
+    formData.append('file', data.file.rawFile);
+
+    fetch('/api/cards/upload', {
+      method: 'POST',
+      body: formData,
+    }).then((response) => {
+      if (response.status === 201) {
+        window.location.href = '/admin/cards';
+      }
+    });
+  };
```

Comment:
Would've been nice to create request hooks for consistency with the other frontend, personally I don't mind but Stijn might yell at you

Reply:
Good comment @RaresGIT. It would be good to keep some consistancy, however since the react admin also handles its own requests, It's not that big of a deal.